server/app/views/components/LinkElement.java

```diff
@@ -84,6 +84,14 @@ public final class LinkElement {
   private boolean doesOpenInNewTab = false;
   private Optional<Icons> icon = Optional.empty();
 
+  public enum IconPosition {
+    NONE,
```

Comment:
Why do we need NONE value?

Reply:
It's a good practice to have unset state for enums. Renamed to UNSET to not be confused with visibility none.

Reply:
I think you are referring to recommendations for enums in proto files? And those are dictated by backward compatibility and over-wire transmission (basically that 0 should be set to UNSET/UNSPECIFIED/NONE). I don't think it applies here where there is no mapping to actual number and this enum never leaves java code. If it ever needs to be transmitted we'll have to consider anyway how to serialize properly.